services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -230,3 +233,21 @@ async def service_retrieve_data_on_ports(
 
         # validate and return
         return RetrieveDataOutEnveloped.parse_obj(response.json())
+
+
+@router.post(
+    "/{node_uuid}:restart",
+    summary="Calls the dynamic service's restart containers endpoint",
+    responses={204: {"model": None}},
```

Comment:
```suggestion
status_code=status.HTTP_204_NO_CONTENT,
```

this is enough

Reply:
do you need to keep ``responses``?? and 204 and status.HTTP_204_NO_CONTENT?

Reply:
only the status_code apparently is required.